_tests_/pet.test.js

```diff
@@ -0,0 +1,155 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet()).toBeInstanceOf(Object);
+  });
+});
+
+describe('constructor', () => {
+  it('sets the name property', () => {
+    const pet = new Pet('fido');
+
+    expect(pet.name).toEqual('fido');
+    expect(pet.hunger).toEqual(0);
+    expect(pet.fitness).toEqual(10);
+  });
+});
+
+describe('growUp', () => {
+  it('has a initial age of 0', () => {
+    const pet = new Pet('fido');
+
+    expect(pet.age).toEqual(0);
+  });
+});
+
+describe('growUp and getting unhealthier', () => {
+  it('increases the age of our pet by 1', () => {
+    const pet = new Pet('fido');
+    pet.growUp();
+
+    expect(pet.age).toEqual(1);
+    expect(pet.hunger).toEqual(5);
+    expect(pet.fitness).toEqual(7);
+  });
+
+  it("growup does not function if pet is not alive" ,() => {
+    const pet = new Pet("fido");
+    pet.age = 30;
+    expect(() => pet.growUp()).toThrow("your pet is no longer alive :(");
+  });
+});
+
+describe('walk', () => {
+  it('increases fitness by 4 to a maximum of 10', () => {
+    const pet = new Pet('fido');
+
+    pet.fitness = 8;
+    pet.walk();
+
+    expect(pet.fitness).toEqual(10);
+  });
+  it("walk does not function if pet is not alive" ,() => {
+    const pet = new Pet("fido");
+    pet.fitness = -1;
+    expect(() => pet.walk()).toThrow("your pet is no longer alive :(");
+  });
+});
+
+describe('feed', () => {
+  it('decreases the pets hunger levels', () => {
+    const pet = new Pet('fido');
+
+    pet.hunger = 0;
+    pet.feed();
+
+    expect(pet.hunger).toEqual(0);
+  });
```

Comment:
In this test you've set the hunger to `0` before running `feed`. As with walk, you should have 2 tests. 1 to show that feed decreases hunger by 3 and on to show that hunger cannot go below 0. Use the examples I gave for `walk` to try and figure out how to write these tests for `feed`